src/main/java/com/softserveinc/ita/pageobjects/HomePage.java

```diff
@@ -29,4 +36,19 @@ public ProductPage openLastProductOfSection(String sectionName) {
 
         return new ProductPage();
     }
+
+    public HomePage openSocialMediaPage(String mediaName) {
+        $x(format("//a[contains(@class, 'socials__link') and @title = '%s']", mediaName)).click();
+
+        return this;
+    }
+
+    public boolean isSocialMediaCorrect(String mediaName) {
```

Comment:
based on comments here, this method not needed, most of these should be performed in test

Reply:
if this method is no longer used, it could be deleted